404.html

```diff
@@ -179,12 +179,22 @@ <h1>TAKE A MULLIGAN</h1>
     </main>
     <footer></footer>
     <script type="module">
-      import { createOptimizedPicture } from "./scripts/lib-franklin.js";
-      const imageUrl = "/media_10b8f0c21a329ee5ab764abf34d64a94de757ea2a.png";
-      const imageAlt = "Tiger Woods regretting a bad shot";
-      const pictureElement = createOptimizedPicture(imageUrl, imageAlt);
-      const imageContainer = document.querySelector(".error-image");
-      imageContainer.appendChild(pictureElement);
+      import { createOptimizedPicture } from "/scripts/lib-franklin.js";
+      import sitemap from "/import/tools/importer/data/sitemap.json" assert {type: 'json'}
```

Comment:
That doesn't work on firefox and other browsers see https://caniuse.com/mdn-javascript_statements_import_import_assertions_type_json

Reply:
firefox should now be supported